pkg/datasources/databases.go

```diff
@@ -4,52 +4,77 @@ import (
 	"context"
 
 	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/internal/provider"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 
 	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
 var databasesSchema = map[string]*schema.Schema{
-	"terse": {
+	"with_describe": {
```

Comment:
what happens now if we flip this flag? (will the state be cleared or will the results stay there?)

Reply:
As every computed value in data source it refreshed (meaning, it made the description and parameters fields empty). Added test case in acc test for confirmation.